learnhubmk/app
# [FE] Content-Panel: Blogs: General Improvements

## Summary of Changes

*   adjusted the design in blogs page to match the tags page
*   added skeletons while fetching data and improved the message UI if no data is available
*   translated all fields (both from Tags and Blogs) to Macedonian
*   improved table UI design
*   removed unnecessary /dashboard route

## Manual Test Steps

1.  Go to /content-panel/tags or /content-panel/blogs and observe loading state and "no data" message. 
2.  Observe the similarity in the designs between those two routes (tags - blogs)

Both pages should be mobile responsive.

# Screenshots

![](https://github.com/user-attachments/assets/931a42ed-6832-44f7-86af-fce8c150786f)

![](https://github.com/user-attachments/assets/2a7c39c2-a30d-4898-baf1-510ee39f9c53)

![](https://github.com/user-attachments/assets/9010c919-62a0-4077-bffb-f44574c11bb4)

![](https://github.com/user-attachments/assets/e33ef1f6-b3db-4ff9-abfa-acaae114a143)

![](https://github.com/user-attachments/assets/704f73dc-32d7-424b-85da-5ef5292a0498)

![](https://github.com/user-attachments/assets/47200c4b-f057-4064-91d3-90c6aab3c323)
